src/components/App.js

```diff
@@ -72,6 +72,7 @@ class App extends Component {
   renderVerticalLayout: Function;
   toggleSymbolModal: Function;
   onEscape: Function;
+  onCommandSlash: Function;
```

Comment:
Someone mentioned that there is another command for opening the shortcuts modal... perhaps we should double check before doing this?  @jasonLaster thoughts? maybe this would be better in a separate pr?